packages/backend/src/database/migrations/20240604140542_add_missing_indexes_for_dashboards_query.ts

```diff
@@ -0,0 +1,46 @@
+import { Knex } from 'knex';
+
+const newTableSingularIndexes = {
+    dashboard_versions: ['dashboard_id', 'updated_by_user_uuid'],
+    spaces: ['project_id'],
+    projects: ['organization_id'],
+    pinned_dashboard: ['pinned_list_uuid', 'dashboard_uuid'],
+    dashboard_tiles: ['dashboard_tile_uuid', 'dashboard_version_id'],
```

Comment:
https://github.com/lightdash/lightdash/blob/67db14c6c9601a984c2aa7643b4dca501f207a50/packages/backend/src/models/DashboardModel/DashboardModel.ts#L350

https://github.com/lightdash/lightdash/blob/67db14c6c9601a984c2aa7643b4dca501f207a50/packages/backend/src/models/DashboardModel/DashboardModel.ts#L344

Reply:
> are you sure there was a seq scan for this?

Couldn't confirm it in the SQL insights as queries are truncated and don't include the filters, and this filter is applied conditionally. Applied this change based on the filter and existing indexes. 
going to remove these individual indexes and update the join.